modules/microshift-lvms-csi-snapshot-optional.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-lvms-csi-snapshot-optional_{context}"]
+= Making LVMS and CSI volume snapshot deployment optional
+
+{microshift-short} now offers optional deployment of the built in logical volume manager storage(LVMS) Container Storage Interface(CSI) provider and CSI volume snapshot capabilities, reducing the use of runtime resources like RAM, CPU and storage.
+
+[id="lvms-stop-deployment_{context}"]
+== Stopping LVMS and CSI volume snapshot deployment
+
+Use the following procedure to prevent the deployment of LVMS CSI driver and CSI volume snapshot controller.
```

Comment:
@dfitzmau By default microshift deploys the LVMS and CSI. If a user does not want to deploy these then the mentioned changes has to be made in config file to avoid the default deployment.
@copejon please confirm "if one deployment runs on both capabilities or just one?"

Reply:
See my comment on the latest version of this line [here](https://github.com/openshift/openshift-docs/pull/80823/files#r1742361104).

See my [comment](https://github.com/openshift/openshift-docs/pull/80823#discussion_r1742338359) re: "deployment".

> Why would a customer want to stop the deployment?

Reply:
the sentence is changed as per latest comments